Travel Scheduler/View Controllers/Place.m

```diff
@@ -238,10 +233,13 @@ - (NSMutableArray *)getRestaurantsPeriodsArrayFromOpeningTime:(float)openingTime
 }
 
 #pragma mark - Methods to get the array of nearby places (for hubs only)
-- (void)createDictionaryOfArrays
+- (void)createDictionaryOfArraysWithHud:(GIFProgressHUD *)hud forView:(UIView *)view
 {
     NSArray *arrayOfTypes = [[NSArray alloc]initWithObjects:@"lodging", @"restaurant", @"museum", @"park", nil];
```

Comment:
I wanted to use enums here, but I do not know if it is worth it since I would have to convert it to a string afterwards to make the API request